Project-Aurora/Project-Aurora/Settings/Configuration.cs

```diff
@@ -440,6 +440,10 @@ public class Configuration : Settings
         public VolumeOverlaySettings volume_overlay_settings;
         public SkypeOverlaySettings skype_overlay_settings;
 
+        //Debug Settings
+        public bool BitmapTopMost;
+        public bool GSIHttpDebugTopMost;
```

Comment:
These should be set up as properties that work with INotifyPropertyChanged (see the HighPriority property)

Reply:
it now uses propertys.